src/persist-client/src/internal/machine.rs

```diff
@@ -1120,13 +1126,14 @@ pub(crate) const NEXT_LISTEN_BATCH_RETRYER_MULTIPLIER: Config<u32> = Config::new
 
 pub(crate) const NEXT_LISTEN_BATCH_RETRYER_CLAMP: Config<Duration> = Config::new(
     "persist_next_listen_batch_retryer_clamp",
-    Duration::from_millis(100), // pubsub is on by default!
+    Duration::from_secs(1), // pubsub is on by default!
```

Comment:
This is just me updated the default to what is actually current in prod (it changed since the 100ms was copied in). Once this PR rolls out, I think we could change this to 16s (or some other large number)